sim/druid/item_sets_pve.go

```diff
@@ -683,11 +757,50 @@ var ItemSetGenesisFury = core.NewItemSet(core.ItemSet{
 	Bonuses: map[int32]core.ApplyEffect{
 		// Each time you Dodge while in Dire Bear Form, you gain 10% increased damage on your next Mangle or Swipe, stacking up to 5 times.
 		2: func(agent core.Agent) {
-			// TODO Bear
+			druid := agent.(DruidAgent).GetDruid()
+
+			druid.Tank2PieceAqProcAura = druid.RegisterAura(core.Aura{
+				Label:     "Guardian 2P Bonus Proc",
+				Duration:  time.Second * 10,
+				MaxStacks: 5,
+				OnStacksChange: func(aura *core.Aura, sim *core.Simulation, oldStacks, newStacks int32) {
+					druid.MangleBear.DamageMultiplierAdditive += 0.1 * float64(newStacks-oldStacks)
+					druid.SwipeBear.DamageMultiplierAdditive += 0.1 * float64(newStacks-oldStacks)
+				},
+			})
+
+			druid.Tank2PieceAqAura = druid.RegisterAura(core.Aura{
+				Label:           "S03 - Item - TAQ - Druid - Guardian 2P Bonus",
+				ActionID:        core.ActionID{SpellID: 1213188},
+				ActionIDForProc: core.ActionID{SpellID: 1213190},
```

Comment:
I'm not 100% sure what `ActionIDForProc` is used for but generally we include the action ID on the proc aura (the one above this one) so that users can track that aura on the timeline / APL / buffs tab then leave it empty on a trigger aura like this unless there's a need to make APL decisions based on having the bonus